src/loader.js

```diff
@@ -1,13 +1,10 @@
-import React, { Component } from "react";
+import React from "react";
 
-class Loader extends Component {
-  render() {
-    if (this.props.showLoader) {
-      return <div className="loader" />;
-    } else {
-      return null;
-    }
+const Loader = (props) => {
```

Comment:
If you have components without state or lifecycle methods, you can write them as functional components instead.

It's nice to use them when you can, because:

there's less code to write
you immediately can tell that it's a simple presentational component
We only briefly covered them before, but they are super easy to write. Just define a function that takes props and returns JSX